aws-ecsfargate-terraform/variables.tf

```diff
@@ -1,15 +1,41 @@
 variable "aws_region" {
   type        = string
-  description = ""
-  default     = "us-east-1"
+  description = "The region of the AWS account where the SCIM bridge will be deployed."
 }
 
 variable "domain_name" {
   type        = string
-  description = ""
+  description = "The public DNS address pointing to your 1Password SCIM bridge."
 }
 
-variable "route53_zone_id" {
+variable "tags" {
+  type        = map(string)
+  description = "A set of tags to apply to all respective AWS resources."
+}
+
+variable "name_prefix" {
+  type        = string
+  description = "A default prefix to apply to the names of all AWS resources."
```

Comment:
My bad. I fixed this in the README by using "common" instead of "default", but I mean some string of text to prepend to the name of each resource. Essentially, the default is nothing, and they just get named how we name them in the Terraform script. If they want to, they can use this variable to tack something else onto the front of those names.